tracks/rust/exercises/bob/mentoring.md

```diff
@@ -88,8 +88,7 @@ match your_trimmed_variable.is_empty() {
     true => FINE,
     false => {
 ```
-Be aware that if you use Clippy, it will tell you that it is improper, it's purely a stylistic choice :)
-We can tell Clippy to allow it with `#[allow(clippy::match_bool)]`.
+Be aware that if you use Clippy, it will tell you that it is improper, it's purely a stylistic choice. We can tell Clippy to allow it with `#[allow(clippy::match_bool)]`.
```

Comment:
Not positive, but one line per sentence seems to work well, if we have adopted that sentiment in this repository, that would be the suggested change here.